### DISCUSSION TITLE
sqlite file in docker [Apr 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1818] [Discord Threads]
> 🙋‍♂️ **PutzMeinScuf**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### u can mount your local path to /usr/app/data/  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/963592297570791528)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
